elastic/src/main/scala/Elastic.scala

```diff
@@ -46,6 +48,18 @@ object Elastic {
 
   private[this] val log = Logger[Elastic.type]
 
+  private[funnel] def daemonThreads(name: String) = new ThreadFactory {
```

Comment:
I'm not sure you need to duplicate this here; can instead do: `Monitoring.daemonThreads`